app/src/main/java/com/google/maps/android/compose/MapSampleActivity.kt

```diff
@@ -127,9 +130,16 @@ private fun GoogleMapView(modifier: Modifier, onMapLoaded: () -> Unit) {
             false
         }
         MarkerInfoWindowContent(
-            position = singapore,
+            position = markerPositionState,
             title = "Zoom in has been tapped $ticker times.",
             onClick = markerClick,
+            draggable = true,
+            onMarkerDrag = { marker, dragState ->
+                markerPositionState = marker.position
```

Comment:
If this line was removed `markerPositionState` will be out-of-sync with the marker's actual position. I don't think there's really anyway around this given how the marker's position is managed by the map. Hoisting a state object as I mentioned in https://github.com/googlemaps/android-maps-compose/pull/11#discussion_r802972785 may be the only appropriate solution here.

Reply:
Ah, I'm sorry. I misunderstood and I agree with you. I'll fix it.